### PR TITLE
docs: include HTTP Transport page in navigation, add support page

### PR DESCRIPTION
**Pull request by chingor13**

### CODE REVIEW

File: docs/support.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Support
+---
+
+# Support
+
+## The Google HTTP Client Library for Java community
```
Comment (elharo) [inline docs/support.md]: Community (caps)

File: docs/support.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Ask development questions
+
+Ask  questions on StackOverflow:
+
+* Use the [google-api][so-google-api] tag, and optionally [java][so-java] or [android][so-android]
```
Comment (elharo) [inline docs/support.md]: is go-java-gae correct or should it be so-java-gae?

File: docs/support.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  or [java google-app-engine][go-java-gae].
+* You can also use the [google-http-java-client][so-http-client] tag.
```
Comment (elharo) [inline docs/support.md]: go/so

Comment (chingor13) [reply docs/support.md]: This one is correct. `so` is for Stack Overflow

**Pull request merged**